x-pack/legacy/plugins/apm/public/components/shared/Summary/HttpInfoSummaryItem/index.tsx

```diff
@@ -83,7 +77,7 @@ export function HttpInfo({ transaction, url }: HttpInfoProps) {
     <Span>
       <HttpInfoBadge title={undefined}>
         <EuiToolTip content={methodLabel}>
-          <strong>{method}</strong>
+          <>{method.toUpperCase()}</>
```

Comment:
This was bold in the designs previously. Did we decide to change it?

Reply:
@smith @katrin-freihofner I decided to remove the `strong` because it was rendering the label a little too strong IMO. I suppose I get that we wanted to differentiate between request method and URL, but is it really that important?

Reply:
I think I originally suggested it, to differentiate it from the URL. I'm okay with removing it though